crates/astria-bridge-withdrawer/src/withdrawer/ethereum/convert.rs

```diff
@@ -95,7 +95,7 @@ fn event_to_bridge_unlock(
         transaction_hash,
     };
     let action = BridgeUnlockAction {
-        to: event.sender.to_fixed_bytes().into(),
+        to: event.destination_chain_address.to_fixed_bytes().into(),
```

Comment:
Should we do validation on the length like we do with the sender for return address with ICS withdrawals?